migrations/versions/89a5b836b406_.py

```diff
@@ -0,0 +1,37 @@
+"""empty message
+
+Revision ID: 89a5b836b406
+Revises: c517c6be040d
+Create Date: 2019-06-06 13:08:53.755940
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '89a5b836b406'
+down_revision = 'c517c6be040d'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute(
+        """DELETE FROM role_invites R1
+        USING role_invites R2
+        WHERE R1.id < R2.id
+        AND R1.email = R2.email
+        AND R1.role_id = R2.role_id
+        AND R1.event_id = R2.event_id
```

Comment:
Uhh, what?

Reply:
Removing duplicates, right? 

So are there any foreign keys in role_invites?

Reply:
I'm deleting all the past duplicate entries here and tested on my local too. When I didn't add this delete condition the upgradation raised error due to presence of duplicate entries. Is the delete statement wrong? @iamareebjamal

Reply:
Yes @iamareebjamal. `event_id` and `role_id` are foreignKeys
https://github.com/fossasia/open-event-server/blob/3b6632114d0214b10b4467cb79d99ed55704aab0/app/models/role_invite.py#L23

Reply:
No, which table has role_invites as foreign key?

Reply:
> I'm deleting all the past duplicate entries here and tested on my local too

Problem is about data loss in production DB, if one of the deleted role invite is linked to other table

Reply:
@iamareebjamal I understand your concern. I've cross-checked and didn't find any model which uses `role_invite` as a foreign key.

Reply:
OK, so reverse the condition `R1.id < R2.id`, we want to delete all but first

Reply:
@iamareebjamal changed the condition.